Docker image build issue

The Dockerfile located at docker/library/dockers/dev-php-fpm-8-5/Dockerfile in the OpenEMR repository fails when installing the PHP ldap extension using [install-php-extensions](https://github.com/mlocati/docker-php-extension-installer).
The error is due to missing lber and ldap pkg-config files during build time, which results in the following:

`configure: error: Package requirements (lber ldap) were not met:
No package 'lber' found
No package 'ldap' found
`

To reproduce this issue  : [github.com/openemr/openemr](https://github.com/openemr/openemr/blob/master/docker/library/dockers/dev-php-fpm-8-5/Dockerfile)
[docker/library/dockers/dev-php-fpm-8-5/Dockerfile](https://github.com/openemr/openemr/blob/master/docker/library/dockers/dev-php-fpm-8-5/Dockerfile)

--- a/docker/library/dockers/dev-php-fpm-8-5-redis/Dockerfile
+++ b/docker/library/dockers/dev-php-fpm-8-5-redis/Dockerfile
@@ -26,11 +26,26 @@ RUN apt-get install -y mariadb-client \
 RUN curl -sL https://deb.nodesource.com/setup_20.x  | bash -
 RUN apt-get install nodejs -y
 
+# Temporary fix to get ldap install working by bypassing below install-php-extensions run (these issues are usually temporary
+#  in dev versions of PHP, so this is not a permanent solution)
+# TODO:
+# TODO: intermittently try to remove this block of code and uncomment ldap in below ../install-php-extensions run
+# TODO:
+RUN apt-get install -y \
+    libldap2-dev \
+    libsasl2-dev \
+    libldap-common \
+    pkg-config
+RUN LDAP_CFLAGS="-I/usr/include" \
+    LDAP_LIBS="-lldap -llber" \
+    docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+    && docker-php-ext-install ldap
+
 # Add the php extensions (note using a very cool script by mlocati to do this)
 ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
 RUN chmod uga+x /usr/local/bin/install-php-extensions && sync && \
     install-php-extensions pdo_mysql \
-                           ldap \
+                           # ldap \
                            xsl \
                            gd \
                            zip \

--- a/docker/library/dockers/dev-php-fpm-8-5/Dockerfile
+++ b/docker/library/dockers/dev-php-fpm-8-5/Dockerfile
@@ -26,11 +26,26 @@ RUN apt-get install -y mariadb-client \
 RUN curl -sL https://deb.nodesource.com/setup_20.x  | bash -
 RUN apt-get install nodejs -y
 
+# Temporary fix to get ldap install working by bypassing below install-php-extensions run (these issues are usually temporary
+#  in dev versions of PHP, so this is not a permanent solution)
+# TODO:
+# TODO: intermittently try to remove this block of code and uncomment ldap in below ../install-php-extensions run
+# TODO:
+RUN apt-get install -y \
+    libldap2-dev \
+    libsasl2-dev \
+    libldap-common \
+    pkg-config
+RUN LDAP_CFLAGS="-I/usr/include" \
+    LDAP_LIBS="-lldap -llber" \
+    docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+    && docker-php-ext-install ldap
+
 # Add the php extensions (note using a very cool script by mlocati to do this)
 ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
 RUN chmod uga+x /usr/local/bin/install-php-extensions && sync && \
     install-php-extensions pdo_mysql \
-                           ldap \
+                           # ldap \
                            xsl \
                            gd \
                            zip \

